.gitignore

```diff
@@ -9,4 +9,7 @@ lib/
 /tmp
 shard.lock
 package-lock.json
-node_modules
\ No newline at end of file
+node_modules
+lib
```

Comment:
lib already ignored

Reply:
I had a small changed, ignore `lib` directory too when it's symbolic link.